src/test/pages/StudentRegistration.test.tsx

```diff
@@ -140,5 +137,28 @@ describe("Student Registration", () => {
     expect(submitButton()).toBeDisabled();
     await userEvent.click(recaptcha());
     expect(submitButton()).toBeEnabled();
-  })
+  });
+
+  it("Clicking the show password icon will make the text of password field visible, and the icon change", async () => {
+    renderStudentRegistration();
+    checkPageTitle("Register as a student");
+    const formFields = getFormFields();
+    const passwordInput = formFields.password() as HTMLInputElement;
+    const confirmPasswordInput =
+      formFields.confirmPassword() as HTMLInputElement;
+    [passwordInput, confirmPasswordInput].forEach((input) => {
+      expect(input.type).toBe("password");
+    });
```

Comment:
Extract the repeated code and create a separate function. That we can then call.

Reply:
Done 👍